cypress/e2e/how-should-replacement-attorneys-step-in.cy.js

```diff
@@ -0,0 +1,39 @@
+describe('How should replacement attorneys step in', () => {
```

Comment:
I think it would be cool after this to rework all of the attorney tests to do the journey each time for each scenario, just because the pages are very interlinked in specific ways.

(And we have weird edge cases that haven't really been thought of; like going through with one attorney and then going back to add another could invalidate what was selected later, but you'd be able to skip back to the task list and leave it in a weird state)

Reply:
Totally agree - once we've got a really clear idea of the different states of the section and the business logic behind it we can merge a bunch of these to give us some confidence.